test/util.test.ts

```diff
@@ -180,46 +177,27 @@ test('should randomly select a reviewer from each group', () => {
     }
     const numberOfReviewers = 2
 
-    //WHEN 
+    // WHEN
     const list = chooseUsersFromGroups(owner, reviewers, numberOfReviewers)
 
-    //THEN
+    // THEN
     expect(list.length).toEqual(1)
     expect(list).toEqual(['reviewer1'])
   })
 
-
-  test('should self assign the owner', () => {
```

Comment:
Even if the reviewers list has only the owner, the owner will not be chosen. Changed at https://github.com/kentaro-m/auto-assign/pull/72